rasa/cli/utils.py

```diff
@@ -203,3 +205,38 @@ def payload_from_button_question(button_question: "Question") -> Text:
 def signal_handler(sig, frame) -> NoReturn:
     print("Goodbye 👋")
     sys.exit(0)
+
+
+def run_in_loop(f: Awaitable[T], loop: Optional[asyncio.AbstractEventLoop] = None) -> T:
```

Comment:
optional: this function is used from a couple of functions outside the `cli` package. This makes me wonder if `rasa.cli.utils` is really the best place for this to live.

Reply:
moved it to common